build.sbt

```diff
@@ -64,7 +64,8 @@ lazy val benchmarks =
         "dev.profunktor"    %% "redis4cats-effects"  % "1.2.0",
         "io.chrisdavenport" %% "rediculous"          % "0.4.0",
         "io.laserdisc"      %% "laserdisc-fs2"       % "0.5.0",
-        "dev.zio"           %% "zio-schema-protobuf" % "0.2.1"
+        "dev.zio"           %% "zio-schema-protobuf" % "0.2.1",
+        "org.redisson"       % "redisson"            % "3.16.8"
```

Comment:
I would not introduce it here. If you want to compare with redisson (which is a debatable idea), I'd do it in a follow-up PR.

Reply:
Sorry. I forgot to delete it. Just use it to explore redisson implementation of cluster.